Makefile

```diff
@@ -52,8 +52,10 @@ export ARCH CC AR LD RM srcdir objdir LDFLAGS
 
 COMMON_CFLAGS := -O2 -g -D_GNU_SOURCE $(CFLAGS) $(CPPFLAGS)
 COMMON_CFLAGS +=  -iquote $(srcdir) -iquote $(objdir) -iquote $(srcdir)/arch/$(ARCH)
+COMMON_CFLAGS +=  -I$(prefix)/include
 #CFLAGS-DEBUG = -g -D_GNU_SOURCE $(CFLAGS_$@)
-COMMON_LDFLAGS := -lelf -lrt -ldl -pthread $(LDFLAGS)
+COMMON_LDFLAGS := -L$(prefix)/lib
+COMMON_LDFLAGS += -lelf -lrt -ldl -pthread $(LDFLAGS)
```

Comment:
Please minimize the diff by adding "-L" option later.

Reply:
done!